logrotate/src/LogRotateHelper.ipp

```diff
@@ -403,9 +427,9 @@ bool LogRotateHelper::createCompressedFile(std::string file_name, std::string gz
         return false;
     }
 
-    int N;
+    size_t N;
     while ((N = fread(buffer, 1, buffer_size, input)) > 0) {
-        gzwrite(output, buffer, N);
+        gzwrite(output, buffer, static_cast<unsigned int>(N));
```

Comment:
Some delay on that as we are putting out fires at work...

Also. I need to install boost on my borrowed Windows machine